BookingsApi/BookingsApi.Infrastructure.Services/Publishers/HearingNotificationEventForJudiciaryParticipantPublisher.cs

```diff
@@ -19,7 +20,17 @@ public HearingNotificationEventForJudiciaryParticipantPublisher(IEventPublisher
 
         public async Task PublishAsync(VideoHearing videoHearing)
         {
+            // we need to send a hearing confirmation for new Panel Member created for V1 and send new templates. A create email for those users
+            // has been sent previously with login details and needs a second email for hearing confirmation
+            var newJudicialOfficers = PublisherHelper.GetNewParticipantsSinceLastUpdate(videoHearing).Where(x => x is JudicialOfficeHolder);
```

Comment:
Given that this publisher is for judiciary participants, I don't think this is in the right place

Reply:
can we add an API integration test for the original issue?